src/AppInstallerCommonCore/Public/winget/ManifestCommon.h

```diff
@@ -184,7 +184,7 @@ bool HasExtension(std::string_view extension) const;
 
         bool operator <(const Dependency& rhs) const
         {
-            return Id < rhs.Id;
+            return FoldCase(Id) < FoldCase(rhs.Id);

```

Comment:
>FoldCase [](http://example.com/codeflow?start=19&length=8)

This is a somewhat expensive function on an operator that needs to be used a lot for sorted containers.  Please have:
```
private:
std::string m_foldedId;
```
And use it in both operators to improve the performance.